pages/linux/zramctl.md

```diff
@@ -0,0 +1,23 @@
+# zramctl
+
+> Setup and control zram devices.
+
+- Check if zram is enabled:
+
+`lsmod | grep -i zram`
+
+- Enable zram with 2 devices:
```

Comment:
> What does "device" mean here exactly ?

By "device" I meant "linux device".

> IIUC, zram allows a sort of extended virtual memory to use RAM further and prevent paging to disk.

Sort of. You can create virtual disks that are stored compressed in RAM, which you can then format as a swap disk with `mkswap`. With that you can enable the new swap partition with a higher priority than any other swap partitions you may already have.